src/assets/sass/protocol/components/_hero.scss

```diff
@@ -0,0 +1,259 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../includes/lib';
+
+// Page hero section - should usually appear at the top of a page
+
+// - The content is centered by default without a hero image.
+// - Add the 'mzp-has-image' class to adjust the layout for a hero image (body left, image right).
+// - Add the 'mzp-l-reverse' class to reverse the layout (image left, body right).
+// - With an image, content is still centered in small viewports, with the image below the body.
+// - Add the 'mzp-t-firefox' class for Firefox branding (changing font to Open Sans).
+// - Add the 'mzp-t-dark' class to invert text colors for dark backgrounds.
+
+.mzp-c-hero {
+    padding-bottom: $spacing-lg;
+    position: relative;
+    text-align: center;
+
+    // Product logos
+    &.mzp-t-product-firefox,
+    &.mzp-t-product-beta,
+    &.mzp-t-product-developer,
+    &.mzp-t-product-nightly,
+    &.mzp-t-product-focus {
+        .mzp-c-hero-title {
+            @include background-size(80px 80px);
+            background-position: top center;
+            background-repeat: no-repeat;
+            padding: (80px + $padding-lg) 0 0 0;
+        }
+    }
+
+    // Dark theme
+    &.mzp-t-dark,
+    .mzp-t-dark & {
+        background-color: $color-gray-80;
+        color: $color-white;
+
+        .mzp-c-hero-desc,
+        .mzp-c-hero-tagline {
+            color: transparentize($color-white, .1);
```

Comment:
Thanks for reminding me... I wanted to run this by the design folks. The mockups have this text as pure white on a dark background, but we're using a dark gray on light backgrounds. So I tried a light gray for the dark theme, but when experimenting with different background colors (blue, purple, etc, and the Pocket page has a teal background) a neutral gray just doesn't look very good. So if we wanted this to be not-quite-white I think we should use a slight transparency rather than gray, or else just go with pure white for all text when the background is dark. We can discuss it or maybe the design team already discussed it and chose white.

Reply:
Solid white it shall be.